contracts/TokenHub.sol

```diff
@@ -237,136 +189,15 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
-        } else {
-            return new bytes(0);
-        }
-    }
-
-    function doTransferIn(TransferInSynPackage memory transInSynPkg) internal returns (uint32) {
-        if (transInSynPkg.contractAddr == address(0x0)) {
-            if (block.timestamp > transInSynPkg.expireTime) {
-                return TRANSFER_IN_FAILURE_TIMEOUT;
-            }
-            if (address(this).balance < transInSynPkg.amount) {
-                return TRANSFER_IN_FAILURE_INSUFFICIENT_BALANCE;
-            }
-
-            // BEP-171: Security Enhancement for Cross-Chain Module
-            if (!_checkAndLockTransferIn(transInSynPkg)) {
-                // directly transfer to the recipient
-                (bool success,) =
-                    transInSynPkg.recipient.call{ gas: MAX_GAS_FOR_TRANSFER_BNB, value: transInSynPkg.amount }("");
-                if (!success) {
-                    return TRANSFER_IN_FAILURE_NON_PAYABLE_RECIPIENT;
-                }
-            }
-
-            emit transferInSuccess(transInSynPkg.contractAddr, transInSynPkg.recipient, transInSynPkg.amount);
-            return TRANSFER_IN_SUCCESS;
-        } else {
-            if (block.timestamp > transInSynPkg.expireTime) {
-                return TRANSFER_IN_FAILURE_TIMEOUT;
-            }
-            if (contractAddrToBEP2Symbol[transInSynPkg.contractAddr] != transInSynPkg.bep2TokenSymbol) {
-                return TRANSFER_IN_FAILURE_UNBOUND_TOKEN;
-            }
-            uint256 actualBalance =
-                IBEP20(transInSynPkg.contractAddr).balanceOf{ gas: MAX_GAS_FOR_CALLING_BEP20 }(address(this));
-            if (actualBalance < transInSynPkg.amount) {
-                return TRANSFER_IN_FAILURE_INSUFFICIENT_BALANCE;
-            }
-
-            // BEP-171: Security Enhancement for Cross-Chain Module
-            if (!_checkAndLockTransferIn(transInSynPkg)) {
-                bool success = IBEP20(transInSynPkg.contractAddr).transfer{ gas: MAX_GAS_FOR_CALLING_BEP20 }(
-                    transInSynPkg.recipient, transInSynPkg.amount
-                );
-                if (!success) {
-                    return TRANSFER_IN_FAILURE_UNKNOWN;
-                }
-            }
-
-            emit transferInSuccess(transInSynPkg.contractAddr, transInSynPkg.recipient, transInSynPkg.amount);
-            return TRANSFER_IN_SUCCESS;
-        }
+        revert("deprecated");
     }
 
     // BEP-171: Security Enhancement for Cross-Chain Module
     function setLargeTransferLimit(
         address bep20Token,
         uint256 largeTransferLimit
     ) external onlyTokenOwner(bep20Token) {
-        require(largeTransferLimit > 0, "zero limit not allowed");
-        require(contractAddrToBEP2Symbol[bep20Token] != bytes32(0x00), "not bound");
-        largeTransferLimitMap[bep20Token] = largeTransferLimit;
-
-        emit LargeTransferLimitSet(bep20Token, msg.sender, largeTransferLimit);
+        revert("deprecated");
```

Comment:
let us disable `withdrawUnlockedToken` function,  we will scan all events to ensure no pending unlock funds there before upgrade.

Reply:
fixed